CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
+    - Branch from last tag on `develop`
+    - Feature branches may pick up "older" branches if they are not to complicated
+    - Combined branches must all start from the same tag
```

Comment:
**question**: I'm not sure I get what you mean correctly. May I request an example?

Reply:
Consider there are two "stable" tags on develop. Let's call them "stable-1" and "stable-2".
Once upon a time, a new feature branch was started from "stable" and called "feature-1".
A second feature branch was started and called "feature-2".

Starting from "stable-2" there is yet another feature branch "stable-3".

During development of "feature-2" you are allowed to pick up "feature-1" (i.e., merge it into "feature-2" and stop commiting to "feature-1") because you need functionality or simply want to have it merged faster. This is okay as both start from the same branch.

If you want to integrate the changes from "feature-3" this means that you would first need to merge "feature-3" into "develop" before merging back from develop into your feature branch.

Reply:
> Starting from "stable-2" there is yet another feature branch **"feature-3"**.

⬆️ I guess.

Also, I'm retrospectively not sure about this in theory. In practice, this adheres to @led02's work mainly, and if you can deal with this, that's fine by me, especially during the ramp up to MVP, when there are still a lot of moving parts anyway. I'll stick to branching off `develop` in my own stuff, cos I understand that better.

Reply:
You're right. During writeup of the explanation, I realized there are some more of less obvious caveats to the strategy. So for me, it's fine to leave the last two points out (while it should still be allowed ;)

Reply:
OK sounds like common then anyway. :smile: